src/graphql/event/conn.rs

```diff
@@ -398,6 +410,18 @@ impl BlockListConn {
         StringNumber(self.inner.resp_pkts)
     }
 
+    async fn orig_l2_bytes(&self) -> u64 {
```

Comment:
Could you replace this and other similar ones with `StringNumber`?

Reply:
Done